src/HHbtagKLUBinterface.cc

```diff
@@ -37,7 +37,12 @@ void HHbtagKLUBinterface::SetInputValues(bigTree &theBigTree, std::vector<std::p
   for (unsigned int i = 0; i < jets_and_sortPar.size(); i++)
   {
     unsigned int idx = jets_and_sortPar.at(i).second;
-    TLorentzVector tlv_jet = smearer.getSmearedJetFromIdx(idx, theBigTree);
+
+    // Build jet
+    TLorentzVector tlv_jet(theBigTree.jets_px->at(idx), theBigTree.jets_py->at(idx), theBigTree.jets_pz->at(idx), theBigTree.jets_e->at(idx));
+
+    // Apply the smearing (it's already set to 1 if !isMC or !doSmearing)
```

Comment:
just to be sure: where is it set to 1 if `!isMC`?

Reply:
In the skimmer: the map is filled with 1. if `doSmearing != true` (for data or if you set it to false in the config)